doc/examples/viz_slice.py

```diff
@@ -148,8 +148,85 @@
    :align: center
 
    **Simple slice viewer with an HSV colormap**.
+"""
+
+"""
+Now we would like to add the ability to click on a voxel and show its value
+on a panel in the window. The panel is a widget which requires access to
+different areas of the visualization pipeline and therefore we don't recommend
+using it with ``window.show``. The more appropriate way is to use the
+``ShowManager`` object, which allows accessing the pipline in different areas.
+"""
+
+show_m = window.ShowManager(renderer, size=(1200, 900))
+show_m.initialize()
+
+"""
+We'll start by creating the panel and adding it to the ``ShowManager``
+"""
+
+
+def build_textblock(text):
```

Comment:
Shouldn't make a function called textblock? And add it in viz.actor or viz.ui?

Reply:
Yes, we should add all of this parameter setting to the `TextBlock2D` constructor. But I think that change should come in another PR because it could affect every other place `TextBlock2D` is used. Also, `build_thing()` functions are used elsewhere in the UI examples.

Reply:
In most UI elements, the constructors give you a lot of control over how the UI element looks like. For simple usecases, I have added a default argument to most of these parameters. For `TextBlock2D`, however, this needs to be done. I will make a quick PR for this today.